src/SAWScript/Crucible/JVM/Override.hs

```diff
@@ -723,6 +723,10 @@ learnPointsTo opts sc cc spec prepost pt = do
          realTerm <- liftIO $ scVector sc ety_tm ts
          matchTerm sc cc loc prepost realTerm (ttTerm tt)
 
+    -- If the right-hand-side is 'Nothing', this is indicates a "modifies" declaration,
+    -- which should probably not appear in the pre-state section, and has no effect.
```

Comment:
Should we warn about this?

Reply:
`jvm_modifies` in the pre-state section is now an error, so this can't happen.